internal/pkg/manifest/workload.go

```diff
@@ -399,8 +400,48 @@ type PublishConfig struct {
 
 // Topic represents the configurable options for setting up a SNS Topic.
 type Topic struct {
-	Name *string `yaml:"name"`
-	Type *string `yaml:"type"`
+	Name *string                 `yaml:"name"`
+	Fifo FifoAdvanceConfigOrBool `yaml:"fifo"`
```

Comment:
nit: we may probably want to write it such that letters in `fifo` have consistent case (all upper or all lower) because "fifo" itself is an acronym. So maybe `FIFO FIFOAdvancedConfigOrBool` here.